generators/client/templates/src/main/webapp/app/blocks/config/_compile.config.js

```diff
@@ -5,13 +5,15 @@
         .module('<%=angularAppName%>')
         .config(compileServiceConfig);
 
-    compileServiceConfig.$inject = ['$compileProvider', 'ENV'];
+    compileServiceConfig.$inject = ['$compileProvider'];
 
-    function compileServiceConfig($compileProvider, ENV) {
+    function compileServiceConfig($compileProvider) {
         // disable debug data on prod profile to improve performance
-        if(ENV === 'prod'){
-            $compileProvider.debugInfoEnabled(false);
-        }
+        
+        // TODO: Howto fetch the dynamic profile in this config.... can this be done elsewhere ?
```

Comment:
What  about in the run block ? (or is that too late ?)

Run Blocks

_Run blocks are the closest thing in Angular to the main method. A run block is the code which needs to run to kickstart the application. It is executed after all of the service have been configured and the injector has been created. Run blocks typically contain code which is hard to unit-test, and for this reason should be declared in isolated modules, so that they can be ignored in the unit-tests._